.gitlab-ci.yml

```diff
@@ -36,7 +36,8 @@ variables:
   BUILDAH_IMAGE:                   "quay.io/buildah/stable:v1.27"
   DOCKER_OS:                       "debian:stretch"
   ARCH:                            "x86_64"
-  ZOMBIENET_IMAGE:                 "docker.io/paritytech/zombienet:v1.2.78"
+  ZOMBIENET_IMAGE:                 "docker.io/paritytech/zombienet:v1.3.16"
+  PIPELINE_SCRIPTS_TAG:            "v0.4"
```

Comment:
Can I ask why do you need this old tag here?

Reply:
I am not sure as it is carried over from a while back. Happy to remove it.

Reply:
Yes, remove this please, it might break something